docs/components/autocomplete.md

```
@@ -1,27 +1,28 @@
+import FwbAutocompleteExampleRemote from './autocomplete/examples/FwbAutocompleteExampleRemote.vue'
 import FwbAutocompleteExampleCustomInput from './autocomplete/examples/FwbAutocompleteExampleCustomInput.vue'
-import FwbAutocompleteExampleValidation from './autocomplete/examples/FwbAutocompleteExampleValidation.vue'
-import FwbAutocompleteExampleSize from './autocomplete/examples/FwbAutocompleteExampleSize.vue'
+import FwbAutocompleteExampleStyling from './autocomplete/examples/FwbAutocompleteExampleStyling.vue'
 </script>
 
-# Vue Autocomplete - Flowbite
+# Autocomplete - Flowbite Vue
 
-Get started with the autocomplete component to allow users to search and select from a list of options with real-time filtering and keyboard navigation.
+#### Get started with the autocomplete component to allow users to search and select from a list of options with real-time filtering and keyboard navigation
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)